client/features/auth-pages/SignUpPage.tsx

```diff
@@ -38,7 +38,8 @@ const SignUpPage = () => {
     handleSubmit,
     formState: { errors },
   } = useForm<FormData>({
-    resolver: zodResolver(userSignUpSchema),
+    resolver: zodResolver(userPasswordSchema),
+    // resolver: zodResolver(userSignUpSchema),
```

Comment:
Can you implement the `userEmailSchema` here to validate the email?

Reply:
I just finished up the changes you all pointed out.  Thank you